tests/Form/Type/ImmutableTabsTypeTest.php

```diff
@@ -30,7 +30,8 @@ class ImmutableTabsTypeTest extends TypeTestCase
      */
     public function testBuildForm()
     {
-        $formBuilder = $this->createMock(FormBuilderInterface::class);
+        $formBuilder            = $this->createMock(FormBuilderInterface::class);
+        $immutableArrayTypeMock = $this->createMock(ImmutableArrayType::class);
```

Comment:
I wonder if rather than implement this, it would be "simpler" to use new ImmutableArrayType(); 
WDYT @mremi  ?

Reply:
no need extra package anymore